src/Silk/Commands/Server/Config/ConfigCommands.cs

```diff
@@ -131,14 +131,19 @@ public async Task<IResult> ViewAllAsync()
                .AppendLine($"> Infraction steps: {(config.InfractionSteps.Count is var dictCount and not 0 ? $"{dictCount} steps [See `config view infractions`]" : "Not configured")}")
                .AppendLine($"> Infraction steps (named): {((config.NamedInfractionSteps?.Count ?? 0) is var infNameCount and not 0 ? $"{infNameCount} steps [See `config view infractions`]" : "Not configured")}")
                .AppendLine()
-               .AppendLine($"**Exemptions** | `exemptions`")
+               .AppendLine("**Exemptions** | `exemptions`")
                .AppendLine($"> {(config.Exemptions.Any() ? $"Configured AutoMod Exemptions: {config.Exemptions.Count}" : "Not configured")}")
                .AppendLine()
-               .AppendLine($"**Anti-Phishing** | `phishing`")
+               .AppendLine("**Anti-Phishing** | `phishing`")
                .AppendLine($"> {(config.DetectPhishingLinks ? Emojis.EnabledEmoji : Emojis.DisabledEmoji)} {Emojis.WarningEmoji} Detect Phishing Links")
                .AppendLine($"> {(config.BanSuspiciousUsernames ? Emojis.EnabledEmoji : Emojis.DisabledEmoji)} {Emojis.ScanEmoji} Ban Suspicious Usernames")
                .AppendLine($"> {(config.DeletePhishingLinks ? Emojis.EnabledEmoji : Emojis.DisabledEmoji)} {Emojis.DeleteEmoji} Delete Phishing Links")
-               .AppendLine($"> {(action is not null ? Emojis.EnabledEmoji : Emojis.DisabledEmoji)} {Emojis.WrenchEmoji} Post-detection action: {phishingAction}");
+               .AppendLine($"> {(action is not null ? Emojis.EnabledEmoji : Emojis.DisabledEmoji)} {Emojis.WrenchEmoji} Post-detection action: {phishingAction}")
```

Comment:
AppendFormat could possibly be used here to increase readability. Unsure on performance due to .NET 6's new API's regarding interpolated strings: https://devblogs.microsoft.com/dotnet/string-interpolation-in-c-10-and-net-6/